apps/site/assets/ts/schedule/components/line-diagram/LineDiagramWithStops.tsx

```diff
@@ -39,7 +46,11 @@ const LineDiagramWithStops = (
           !anyCrowding ? "u-no-crowding-data" : ""
         }`}
       >
-        <Diagram lineDiagram={stops} liveData={liveData} />
+        <Diagram
+          lineDiagram={stops}
+          liveData={liveData}
```

Comment:
we can probably stop passing `liveData` here and just use the data from `vehicleMarkers`